src/coresyftools/coresyftools/tests/test_tester.py

```diff
@@ -1,12 +1,28 @@
 from unittest import TestCase
 
 from ..tool_tester import ToolTester, NonZeroReturnCode, NonEmptyStderr, NoOutputFile, EmptyOutputFile
+import os
 
 
 class TestTester(TestCase):
 
+    def test_download_input(self):
+        tester = ToolTester('coresyftools/tests/dummy_tool',
+                            ('rccc', 'ssdh2dme'))
+        tester._download_input('downloaded_file', "https://scihub.copernicus.eu/dhus/odata/v1/Products('139d30eb-bc03-459c-8642-e794aad61707')/$value", tester.scihub_credentials)
```

Comment:
I would suggest creating a global variable with the URL above so that we don't have so many long lines.